export.py

```diff
@@ -1600,8 +1614,11 @@ def export_geometry_data(ri, scene, ob, data=None):
         export_points(ri, ob, data)
 
 
+#this method is weird - why does the parameter recurse indicate whether this is a recursed call? instead of used 
```

Comment:
That is weird.  It might be a vestige of some old code.  I think we can take it out.